regimes/pl/extensions.go

```diff
@@ -9,6 +9,7 @@ import (
 const (
 	ExtKeyKSeFVATZero    = "pl-ksef-vat-zero"
 	ExtKeyKSeFVATSpecial = "pl-ksef-vat-special"
+	ExtKeyKSEFCorrection = "pl-ksef-correction"
```

Comment:
Correction is too generic, I'd propose:
```suggestion
	ExtKeyKSEFEffectiveDate = "pl-ksef-effective-date"
```